### ISSUE TITLE
Yolo v3 Tiny Mask

### ISSUE THREAD

**Issue by felixendres** [pjreddie/darknet#851]
Hi,
similar question to #561
https://github.com/pjreddie/darknet/blob/f6d861736038da22c9eb0739dca84003c5a5e275/cfg/yolov3-tiny.cfg#L175

**Comment by twtygqyy** [comment]
Same question, why the masks are 1,2,3 instead of 0,1,2?

**Comment by senliuy** [comment]
Same question, shouldn't it be 0,1,2?

**Comment by zhangming8** [comment]
Same question, shouldn't it be 0,1,2?

**Comment by TheMikeyR** [comment]
yes it should be 0,1,2, but since the original weights are trained on 1,2,3, you should not change it before @pjreddie adds new weights trained with correct masks or unless you train it yourself from the bottom on a big dataset such as imagenet to get a good generalised model which can act as a starting point for custom datasets.
